src/services/associationsService.ts

```diff
@@ -0,0 +1,121 @@
+import { Request } from "express";
+import { createOauthPrivateApiClient } from "./apiClientService";
+import { Resource } from "@companieshouse/api-sdk-node";
+import logger from "../lib/Logger";
+import { StatusCodes } from "http-status-codes";
+import * as constants from "../constants";
+import { Associations, AssociationStatus, Errors, NewAssociationResponse } from "@companieshouse/private-api-sdk-node/dist/services/associations/types";
+
+export const getUserAssociations = async (req: Request, status: AssociationStatus[]): Promise<Associations> => {
+    const apiClient = createOauthPrivateApiClient(req);
+
+    logger.info(`Looking for associations with status ${JSON.stringify(status)}`);
+    const sdkResponse: Resource<Associations | Errors> = await apiClient.associationsService.searchAssociations(status);
+
+    if (!sdkResponse) {
+        logger.error(`Associations API for status ${JSON.stringify(status)}`);
+        return Promise.reject(sdkResponse);
+    }
```

Comment:
Probably not something we need to handle now but we will need to update when we are working on the unhappy path / error handling.

Normally promise.rejection takes a rejection reason why the promise is rejected or an error. 
In this instance we are just passing undefined as the rejection reason.